addons/ofxOsc/src/ofxOscReceiver.cpp

```diff
@@ -39,8 +39,12 @@ ofxOscReceiver::ofxOscReceiver()
 	listen_socket = NULL;
 }
 
-void ofxOscReceiver::setup( int listen_port )
+void ofxOscReceiver::setup( int listen_port, bool allowReuse )
```

Comment:
where do these unrelated changes come from? they have nothing to do with the PR subject, and that's something that @ofTheo has already merged in, no? I'd be hesitant to touch the reuse code again, tbh.